src/CalcViewModel/Common/Utils.cpp

```diff
@@ -86,11 +86,11 @@ bool Utils::IsLastCharacterTarget(_In_ wstring const &input, _In_ wchar_t target
     return !input.empty() && input.back() == target;
 }
 
-//return wstring after removing characters like space, comma, and double quotes
+//return wstring after removing characters like space, comma, double quotes, and monetary prefix currency symbols supported by the Windows keyboard
 wstring Utils::RemoveUnwantedCharsFromWstring(wstring input)
```

Comment:
This function is only used by `CopyPasteManager`.  Let's remove it from `Utils` and move it to that file.